packages/neon/neon/lib/src/models/push_notification.dart

```diff
@@ -1,28 +1,72 @@
 import 'package:json_annotation/json_annotation.dart';
 import 'package:meta/meta.dart';
-import 'package:nextcloud/notifications.dart' show DecryptedSubject;
+import 'package:nextcloud/notifications.dart' as notifications;
 
 part 'push_notification.g.dart';
 
+/// The json key for [PushNotification.accountID].
+const String _accountIDKey = 'accountID';
+
+/// The json key for [PushNotification.priority].
+const String _priorityKey = 'priority';
+
+/// The json key for [PushNotification.type].
+const String _typeKey = 'type';
+
+/// The json key for [PushNotification.subject].
+const String _subjectKey = 'subject';
```

Comment:
This seems useless now that there is only a single class with those fields.

Reply:
No it isn't.
I want to ensure the same key is used for both the encrypted and decrypted serializaiton.
Therefore it's still used in the ´fromEncrypted` method.

Reply:
Ahhh, I oversaw that.